examples/rotation_bin/README.me

```diff
@@ -0,0 +1,5 @@
+# RotationBin
```

Comment:
The extension of this file is wrong :) also the example should be added to the main README in the examples directory.